_emails/newark.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Newark, NJ
+permalink: "/newark"
+name: Letter to Mayor Baraka and City Councilors
+state: NJ
+city: Newark
+layout: email
+recipients:
+- barakara@ci.newark.nj.us
+- ramosa@ci.newark.nj.us
+- amadora@ci.newark.nj.us
+- gonzalezc@ci.newark.nj.us
+- osbornee@ci.newark.nj.us
+- jamesjoh@ci.newark.nj.us
+- mccallumjo@ci.newark.nj.us
+- mciverl@ci.newark.nj.us
+- quintanal@ci.newark.nj.us
+- crumpm@ci.newark.nj.us
+subject: Newark Demands Equity: Police Budget and Reform
+body: My name is [NAME] and I am a resident of Newark. I urge you to redirect money away from the NPD and into social service programs that will benefit public health and our own communities. We also need to ban chokeholds, strangleholds, shooting at moving vehicles, and require police officers to practice de-escalation and comprehensive reporting (8cantwait.org).
+
+Newark's 2019 Municipal Budget shows that $154,625,090 was spent on the Division of Police. This is by far the largest fraction of the city's budget, which is over double the second largest allocation of funds (Division of Fire's $72,354,451). It's an egregious statistic compared to $9,407,451 spent on the entire Department of Health and Community Wellness, $3,504,842 on the Department of Economic and Housing Development, and just $359,239 to the Division of Cultural Affairs. If this is truly representative of the council’s thoughts on how funds should be allocated in a city in which one in every 3.5 residents live in poverty (see source below), I am ashamed to call Newark my home.
```

Comment:
Body text should be piped and indented, otherwise it won't build.

See an example here:
https://github.com/teddywilson/defund12.org/pull/178/files